src/SFML/Audio/SoundBuffer.cpp

```diff
@@ -34,30 +34,22 @@
 ////////////////////////////////////////////////////////////
 sfSoundBuffer* sfSoundBuffer_createFromFile(const char* filename)
 {
-    sfSoundBuffer* buffer = new sfSoundBuffer;
+    const auto soundBuffer = sf::SoundBuffer::loadFromFile(filename);
```

Comment:
This needs to be mutable to allow being moved from. (assuming SoundBuffer eventually implements move semantics)

Reply:
Sound buffers are already moveable so removing this `const` will enable a move. Thanks. I think clang-tidy will flag this but we don't have clang-tidy set up on CSFML yet.

Reply:
To clarify my point: yes `sf::SoundBuffer` is moveable, but only as a consequence of being copyable. There is no specific move constructor/assignment operator that transfers ownership yet.

Reply:
https://github.com/SFML/SFML/blob/fdcd899d2b027bf61d11e69e44e55e6727ac6205/test/Audio/SoundBuffer.test.cpp#L20-L23

`sf::SoundBuffer` is in fact move constructible and move assignable.